docs/TEAM.md

```diff
@@ -9,7 +9,7 @@
 | <img src="./images/team/eyram.jpg" width="80px" height="80px">                           | [Eyram ](https://twitter.com/Ey_kwasi)                       | I am from Ghana and I have no experience, started learning web development some months ago. I am participating to learn and also make twindle a success.                                                                                                                                                                                                                                                   |
 |                                                                                          | [Michael Osajeh](https://github.com/michaelcosj)             | I am from Lagos, Nigeria, second year Computer Science Undergraduate, started learning web and mobile development this year.                                                                                                                                                                                                                                                                               |
 |                                                                                          | [Tushar Kandpal](https://github.com/tusharkandpal)           | From India, Contributing in making **twindle** a succees, Always a Learner :v:                                                                                                                                                                                                                                                                                                                             |
-| ![avatar](https://github.com/RafaelBatman55.png?size=80) | [Rafael Rodrigues](https://github.com/RafaelBatman55)        | Less than 1 yr of experience From small city in Rio de Janeiro State - Braz.I'm very excited to learn and helf in this project!                                                                                                                                                                                                                                                                            |
+| ![avatar](https://github.com/RafaelBatman55.png?size=80)                                 | [Rafael Rodrigues](https://github.com/RafaelBatman55)        | Less than 1 yr of experience From small city in Rio de Janeiro State - Braz.I'm very excited to learn and helf in this project!                                                                                                                                                                                                                                                                            |
```

Comment:
Why did you add this kind of space after my avatar? Is it necessary to works better?

Reply:
Please, I am using prettier extension for code formatting,  I did not edit your code

Reply:
Please disable the prettier extension for this workspace in VS Code, revert the changes and add your local changes again.

Reply:
Ok, I will do it

Reply:
No problem @samueladeniyi!